sklearn/linear_model/logistic.py

```diff
@@ -1345,8 +1346,8 @@ def fit(self, X, y):
         check_consistent_length(X, y)
 
         # init cross-validation generator
-        cv = check_cv(self.cv, X, y, classifier=True)
-        folds = list(cv)
+        cv = check_cv(self.cv, y, classifier=True)
```

Comment:
well `y` can't simply be removed without more changes (if `y=None` then stratification is not used in `check_cv`). The easiest would be to leave it.